pkg/ccl/storageccl/export_storage.go

```diff
@@ -99,6 +100,14 @@ func ExportStorageConfFromURI(path string) (roachpb.ExportStorage, error) {
 	return conf, nil
 }
 
+func getParamOrEnv(u *url.URL, name string) string {
+	s := u.Query().Get(name)
+	if s != "" {
+		return s
+	}
+	return envutil.EnvOrDefaultString(name, "")
```

Comment:
IIRC, `envutil` wants all calls at init time, and I though it also wanted everything prefixed with COCKROACH_?